config/python-trollius.backport.yaml

```diff
@@ -0,0 +1,6 @@
+name: trollius
+method: http://ppa.launchpad.net/wjwwood/trollius-backports/ubuntu
+suites: [trusty]
+component: main
+architectures: [any]
```

Comment:
apparently in this context neither `any` nor `all` work in this context. I think we're going to have to enumerate the supported arches unfortunately.